File: SelfSchoolBE/SelfSchoolDAL/Entities/ClassSchool.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SelfSchoolDAL.Entities
{
    public class ClassSchool
    {
        [Key]
        public int idClass { get; set; }
        public int numberClass { get; set; }
        public string letterClass { get; set; }
    }
}

File: SelfSchoolFE/src/app/classes/class-school.ts
export class ClassSchool {
  private readonly _idClass: number = 0;
  private readonly _numberClass: number = 0;
  private readonly _letterClass: string = '';

  constructor(idClass: number, numberClass: number, letterClass: string) {
    this._idClass = idClass;
    this._numberClass = numberClass;
    this._letterClass = letterClass;
  }

  public get idClass(): number {
    return this._idClass;
  }

  public get numberClass(): number {
    return this._numberClass;
  }

  public get letterClass(): string {
    return this._letterClass;
  }
}
